examples/app/commands_and_api/.lightningignore

```diff
@@ -0,0 +1 @@
+venv/
```

Comment:
is this used?

Reply:
This is automatically generated when the app runs. Better to have it in the repo.